src/System.Security.Permissions/ref/System.Security.Permissions.cs

```diff
@@ -278,27 +312,30 @@ public sealed partial class FileDialogPermissionAttribute : System.Security.Perm
     public override System.Security.IPermission CreatePermission() { throw null; }
   }
-  public sealed partial class FileIOPermission : System.Security.CodeAccessPermission, System.Security.Permissions.IUnrestrictedPermission {
-    public FileIOPermission(System.Security.Permissions.FileIOPermissionAccess access, string path) { }
-    public FileIOPermission(System.Security.Permissions.FileIOPermissionAccess access, string[] pathList) { }
-    public FileIOPermission(System.Security.Permissions.PermissionState state) { }
-    public System.Security.Permissions.FileIOPermissionAccess AllFiles { get; set; }
-    public System.Security.Permissions.FileIOPermissionAccess AllLocalFiles { get; set; }
-    public void AddPathList(System.Security.Permissions.FileIOPermissionAccess access, string path) { }
-    public void AddPathList(System.Security.Permissions.FileIOPermissionAccess access, string[] pathList) { }
-    public override System.Security.IPermission Copy() { throw null; }
-    public override bool Equals(object o) => base.Equals(o);
-    public override void FromXml(System.Security.SecurityElement esd) { }
-    public override int GetHashCode() => base.GetHashCode();
-    public string[] GetPathList(System.Security.Permissions.FileIOPermissionAccess access) { throw null; }
-    public override System.Security.IPermission Intersect(System.Security.IPermission target) { throw null; }
-    public override bool IsSubsetOf(System.Security.IPermission target) { throw null; }
-    public bool IsUnrestricted() { throw null; }
-    public void SetPathList(System.Security.Permissions.FileIOPermissionAccess access, string path) { }
-    public void SetPathList(System.Security.Permissions.FileIOPermissionAccess access, string[] pathList) { }
-    public override System.Security.SecurityElement ToXml() { throw null; }
-    public override System.Security.IPermission Union(System.Security.IPermission other) { throw null; }
-  }
+  public sealed partial class FileIOPermission : System.Security.CodeAccessPermission, System.Security.Permissions.IUnrestrictedPermission
+    {
+        public FileIOPermission(System.Security.Permissions.FileIOPermissionAccess access, System.Security.AccessControl.AccessControlActions actions, string path) { }
+        public FileIOPermission(System.Security.Permissions.FileIOPermissionAccess access, System.Security.AccessControl.AccessControlActions actions, string[] pathList) { }
+        public FileIOPermission(System.Security.Permissions.FileIOPermissionAccess access, string path) { }
+        public FileIOPermission(System.Security.Permissions.FileIOPermissionAccess access, string[] pathList) { }
+        public FileIOPermission(System.Security.Permissions.PermissionState state) { }
+        public System.Security.Permissions.FileIOPermissionAccess AllFiles { [System.Runtime.CompilerServices.CompilerGeneratedAttribute]get { throw null; } [System.Runtime.CompilerServices.CompilerGeneratedAttribute]set { } }
```

Comment:
I think you can just remove all the CompilerGeneratedAttribute adornments. Nobody is going to debug into this code.